nemo_text_processing/text_normalization/de/taggers/electronic.py

```diff
@@ -35,31 +35,44 @@ def __init__(self, deterministic: bool = True):
         super().__init__(name="electronic", kind="classify", deterministic=deterministic)
 
         dot = pynini.accep(".")
-        accepted_common_domains = [x[0] for x in load_labels(get_abs_path("data/electronic/domain.tsv"))]
-        accepted_common_domains = pynini.union(*accepted_common_domains)
-        accepted_symbols = [x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))]
-        accepted_symbols = pynini.union(*accepted_symbols) - dot
-        accepted_characters = pynini.closure(NEMO_ALPHA | NEMO_DIGIT | accepted_symbols)
+
+        all_symbols = [x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))]
+        all_symbols = pynini.union(*all_symbols)
+        # all symbols
+        accepted_symbols = pynini.difference(all_symbols, dot)  # alphabet of accepted symbols excluding the '.'
+        accepted_characters = pynini.closure(
```

Comment:
Need more specific variable names. `all` vs `accepted` is awkward.

Reply:
Will fix.

Reply:
Fixed

Reply:
Did you push? Not showing up